server/client.go

```diff
@@ -91,6 +92,12 @@ type clientOpts struct {
 	Version       string `json:"version"`
 }
 
+// Event data published when a client connects or disconnects
+type clientConnEvent struct {
```

Comment:
Depends, where do they get the additional information? If its via connz then I think we make sure connz can support connz?cid=N and have the event from gnatsd just send CID as the payload or as part of the subject with no payload.